data/aliases.js

```diff
@@ -826,29 +826,814 @@ let BattleAliases = {
+	"nidoranosu": "Nidoran♂",
+	"nidorino": "Nidorino",
+	"nidokingu": "Nidoking",
+	"pippi": "Clefairy",
+	"pikushi": "Clefable",
+	"rokon": "Vulpix",
+	"kyuukon": "Ninetales",
+	"purin": "Jigglypuff",
+	"pukurin": "Wigglytuff",
+	"zubatto": "Zubat",
+	"gorubatto": "Golbat",
+	"nazonokusa": "Oddish",
+	"kusaihana": "Gloom",
+	"rafureshia": "Vileplume",
+	"parasu": "Paras",
+	"parasekuto": "Parasect",
+	"konpan": "Venonat",
+	"morufon": "Venomoth",
+	"diguda": "Diglett",
+	"dagutorio": "Dugtrio",
+	"nyaasu": "Meowth",
+	"perushian": "Persian",
+	"kodakku": "Psyduck",
+	"gorudakku": "Golduck",
+	"manki": "Mankey",
+	"okorizaru": "Primeape",
+	"gaadi": "Growlithe",
+	"uindi": "Arcanine",
+	"nyoromo": "Poliwag",
+	"nyorozo": "Poliwhirl",
+	"nyorobon": "Poliwrath",
+	"keishi": "Abra",
+	"yungeraa": "Kadabra",
+	"fuudin": "Alakazam",
+	"wanriki": "Machop",
+	"gouriki": "Machoke",
+	"kairiki": "Machamp",
+	"madatsubomi": "Bellsprout",
+	"utsudon": "Weepinbell",
+	"utsubotto": "Victreebel",
+	"menokurage": "Tentacool",
+	"dokukurage": "Tentacruel",
+	"ishitsubute": "Geodude",
+	"goroun": "Graveler",
+	"gorounya": "Golem",
+	"ponita": "Ponyta",
+	"gyaroppu": "Rapidash",
+	"yadon": "Slowpoke",
+	"yadoran": "Slowbro",
+	"koiru": "Magnemite",
+	"reakoiru": "Magneton",
+	"kamonegi": "Farfetch'd",
+	"doudou": "Doduo",
+	"doudorio": "Dodrio",
+	"pauwau": "Seel",
+	"jugon": "Dewgong",
+	"betobetaa": "Grimer",
+	"betobeton": "Muk",
+	"sherudaa": "Shellder",
+	"parushen": "Cloyster",
+	"gousu": "Gastly",
+	"gousuto": "Haunter",
+	"gengaa": "Gengar",
+	"iwaaku": "Onix",
+	"suripu": "Drowzee",
+	"suripaa": "Hypno",
+	"kurabu": "Krabby",
+	"kinguraa": "Kingler",
+	"biriridama": "Voltorb",
+	"marumain": "Electrode",
+	"tamatama": "Exeggcute",
+	"nasshi": "Exeggutor",
+	"karakara": "Cubone",
+	"garagara": "Marowak",
+	"sawamuraa": "Hitmonlee",
+	"ebiwaraa": "Hitmonchan",
+	"beroringa": "Lickitung",
+	"dogaasu": "Koffing",
+	"matadogasu": "Weezing",
+	"saihoun": "Rhyhorn",
+	"saidon": "Rhydon",
+	"rakki": "Chansey",
+	"monjara": "Tangela",
+	"garuura": "Kangaskhan",
+	"tattsuu": "Horsea",
+	"shidora": "Seadra",
+	"tosakinto": "Goldeen",
+	"azumaou": "Seaking",
+	"hitodeman": "Staryu",
+	"sutaami": "Starmie",
+	"bariyaado": "Mr. Mime",
+	"sutoraiku": "Scyther",
+	"ruujura": "Jynx",
+	"erebuu": "Electabuzz",
+	"buubaa": "Magmar",
+	"kairosu": "Pinsir",
+	"kentarosu": "Tauros",
+	"koikingu": "Magikarp",
+	"gyaradosu": "Gyarados",
+	"rapurasu": "Lapras",
+	"metamon": "Ditto",
+	"bui": "Eevee",
+	"shawaazu": "Vaporeon",
+	"sandaasu": "Jolteon",
+	"buusutaa": "Flareon",
+	"porigon": "Porygon",
+	"omunaito": "Omanyte",
+	"omusutaa": "Omastar",
+	"kabuto": "Kabuto",
+	"kabutopusu": "Kabutops",
+	"putera": "Aerodactyl",
+	"kabigon": "Snorlax",
+	"furizaa": "Articuno",
+	"sandaa": "Zapdos",
+	"faiyaa": "Moltres",
+	"miniryuu": "Dratini",
+	"hakuryuu": "Dragonair",
+	"kairyuu": "Dragonite",
+	"myuutsuu": "Mewtwo",
+	"myuu": "Mew",
+	"chikorita": "Chikorita",
```

Comment:
It's because of these entries; it's unsupported for aliases to reference themselves.

Reply:
Thanks, good catch. Updated.